### PR TITLE
feat: Create storage monorepo

### PR DESCRIPTION
**Pull request by elliott-with-the-longest-name-on-github**
Don't worry, `edge-config` is still here! `/packages/edge-config`

### CODE REVIEW

File: .changeset/pretty-lamps-develop.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@vercel/edge-config': patch
```
Comment (dferber90) [inline .changeset/pretty-lamps-develop.md:2]: do we actually need to release a patch for edge-config? no point in bumping the version if we didn't make changes

image

Comment (jaredpalmer) [reply .changeset/pretty-lamps-develop.md:2]: No

Comment (dferber90) [reply .changeset/pretty-lamps-develop.md:2]: haha sorry, wasn't actually asking - it's just my polite way of saying "this should probably not be here"

Comment (elliott-with-the-longest-name-on-github) [reply .changeset/pretty-lamps-develop.md:2]: Actually, we need to publish it to update the GitHub links on NPM -- they'll point to a nonexistent repo if we don't.

File: README.md [hunk 2]
```diff
@@ -1,116 +1,7 @@
-# @vercel/edge-config
+# Vercel Storage
 
-![CI](https://github.com/vercel/edge-config/workflows/CI/badge.svg)
-![Edge Runtime Compatible](https://img.shields.io/badge/edge--runtime-%E2%9C%94%20compatible-black)
+- [`@vercel/postgres`](./packages/postgres) - Vercel Postgres database client
```
Comment (dferber90) [inline README.md:3]: should link to edge config here as well

**Review by jaredpalmer — APPROVED** [review]

**Pull request merged**